packages/app/ui/components/PostContent/BlockRenderer.tsx

```diff
@@ -242,7 +242,11 @@ export function LinkBlock({
   }, [block.url]);
 
   return (
-    <Reference.Frame {...props} onPress={clickable ? onPress : undefined}>
+    <Reference.Frame
+      {...props}
+      maxWidth={400}
```

Comment:
seems like this will override the 600 dan set earlier

Reply:
oop, good catch. I was just toying with it locally, did not mean to commit.